src/module/listClass.js

```diff
@@ -0,0 +1,60 @@
+export default class TodoList {
+  allTodos = [];
+
```

Comment:
Please use let or const to declare the array

Reply:
Hello @Mujeeb4582!
Great job so far, just a few edits to move on to the next project


## Highlights
- No linter errors
- Professional git-flow
- Great design

## Required Changes ♻️

Reply:
@Tamunokuro Thanks for your time and feedback. you suggested me that try to declare an array using let or const but I declare the array using `class`, when you are declaring an array using `class` there is no need of `let` or `const`.